<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stage</class>
 <widget class="QMainWindow" name="Stage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="RIP1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RIP</string>
    </property>
   </widget>
   <widget class="QLabel" name="RIP2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RIP</string>
    </property>
   </widget>
   <widget class="QLabel" name="coffin1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>coffin1</string>
    </property>
   </widget>
   <widget class="QLabel" name="zombie_label_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>39</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>zombie1</string>
    </property>
   </widget>
   <widget class="QLabel" name="ladder_label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>-10</y>
      <width>29</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>laddar</string>
    </property>
   </widget>
   <widget class="QLabel" name="zombie_label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>38</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>zombie2</string>
    </property>
   </widget>
   <widget class="QLabel" name="player_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>48</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>player</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="stage_background_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="InfoVLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="stageHLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Stage:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="level_label">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lifeHLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Life:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="spellHLayout">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Spell:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="RIP3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RIP</string>
    </property>
   </widget>
   <widget class="QLabel" name="fire_label_1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>36</y>
      <width>65</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="player_label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>48</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>player</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fire_label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>475</y>
      <width>65</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fire_label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>475</y>
      <width>65</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>stage_background_label</zorder>
   <zorder>ladder_label</zorder>
   <zorder>player_label</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>coffin1</zorder>
   <zorder>zombie_label_1</zorder>
   <zorder>zombie_label_2</zorder>
   <zorder>RIP1</zorder>
   <zorder>RIP2</zorder>
   <zorder>RIP3</zorder>
   <zorder>fire_label_1</zorder>
   <zorder>player_label_2</zorder>
   <zorder>fire_label_2</zorder>
   <zorder>fire_label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>goUpOneLevel()</slot>
 </slots>
</ui>
